### ISSUE TITLE
bin packing stable problem

### ISSUE THREAD

**Issue by Zhaocka** [enzoruiz/3dbinpacking#34]
Does this algorithm take into account the problem of packing stability? For example, an item may not be supported by another item under it.

**Comment by moschd** [comment]
No, packing stability is not taken into account in this implementation.
